Header header
bool leftContactSwitch
bool rightContactSwitch

